image/src/mobilenet/custom-mobilenet.ts

```diff
@@ -184,13 +204,18 @@ export class CustomMobileNet {
 
 /**
  * load the base mobilenet model
- * @param checkpoint the URL to load the mobilenet json
+ * @param modelOptions options determining what model to load
  */
-export async function loadTruncatedMobileNet(checkpoint: string = MOBILENET_MODEL_PATH) {
-    const mobilenet = await tf.loadLayersModel(checkpoint);
-    const layer = mobilenet.getLayer('conv_pw_13_relu');
-    const truncatedModel = tf.model({ inputs: mobilenet.inputs, outputs: layer.output });
-    return truncatedModel;
+export async function loadTruncatedMobileNet(modelOptions?: ModelOptions) {
+    const [checkpointUrl, trainingLayer] = parseModelOptions(modelOptions);
```

Comment:
`trainingLayer` appears to never be used? and is included in `ModelOptions`, here in this function and in `parseModelOptions` is it used somewhere and I am missing it?